Resources/Prototypes/_NF/Entities/Structures/Machines/lathe.yml

```diff
@@ -1014,6 +1014,11 @@
       - PlushieBeeRecipe
       - PlushieLizardRecipe
       - PlushieSpaceLizardRecipe
+      - Plushie_BotanistRecipe
+      - Plushie_CmoRecipe
+      - Plushie_EngineerRecipe
+      - Plushie_ConstructionRecipe
+      - Plushie_SharkmMinnowRecipe
```

Comment:
Will say that these should probably be in the emag side of things if only for consistency with the other plushies (special callouts to active players seems a bit gauche if it isn't fairly rare or obscure).